main.go

```diff
@@ -186,11 +186,22 @@ func render(tmpl string, vars map[string]string) (string, error) {
 		"hasPrefix": strings.HasPrefix,
 		"hasSuffix": strings.HasSuffix,
 		"split": strings.Split,
-	}
+                "file": func(key string) (string, error) {
+                        data, err := ioutil.ReadFile(key)
+                        if err != nil {
+                                return "" , err
```

Comment:
Its rare to throw and error in a template function ... given the template content is important i'd say it either works or throw's a panic .. In the case return string and above the Execute of the template defer and capture panic, os.Exit(1) with a contextual error .. https://github.com/kubernetes/kops/blob/master/pkg/util/templater/templater.go#L62-L70

Reply:
Also remove the inline function and create a new function out of it